### ISSUE TITLE
nexus_wallet-Linux-1.0.1.deb doesn't work

### ISSUE THREAD

**Issue by pioruns** [Nexusoft/NexusInterface#49]
Just installed **nexus_wallet-Linux-1.0.1.deb** in my Linux system: MX 18.3 Continuum (essentially **Debian Stretch** with different UI). kernel Linux **mx 4.19.0-5-amd64 #1 SMP Debian 4.19.37-2~mx17+1 (2019-05-15) x86_64 GNU/Linux**. Rest is pretty much stock Debian Stretch.
Results:

```
$ nexus_wallet 
path.js:1084
          cwd = process.cwd();
                        ^

Error: ENOENT: no such file or directory, uv_cwd
    at Object.resolve (path.js:1084:25)
    at Function.Module._resolveLookupPaths (internal/modules/cjs/loader.js:482:17)
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:577:20)
    at Function.Module._load (internal/modules/cjs/loader.js:510:25)
    at Module.require (internal/modules/cjs/loader.js:640:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at Object.<anonymous> (/opt/Nexus Wallet/resources/electron.asar/browser/init.js:15:1)
    at Object.<anonymous> (/opt/Nexus Wallet/resources/electron.asar/browser/init.js:190:3)
    at Module._compile (internal/modules/cjs/loader.js:693:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:704:10)
```

That's it, it won't go any further, completely broken.

**Comment by pioruns** [comment]
Just tried to uninstall it from my system. Errors even when unistalling:

```
$ sudo aptitude purge nexus-wallet
[sudo] password for nv01: 
The following packages will be REMOVED:  
  nexus-wallet{p} 
0 packages upgraded, 0 newly installed, 1 to remove and 1 not upgraded.
Need to get 0 B of archives. After unpacking 334 MB will be freed.
Do you want to continue? [Y/n/?] y
sh: 0: getcwd() failed: No such file or directory
sh: 0: getcwd() failed: No such file or directory
sh: 0: getcwd() failed: No such file or directory
sh: 0: getcwd() failed: No such file or directory
(Reading database ... 287571 files and directories currently installed.)
Removing nexus-wallet (1.0.1) ...
Processing triggers for mime-support (3.60) ...
Processing triggers for desktop-file-utils (0.23-1) ...
Processing triggers for hicolor-icon-theme (0.15-1) ...
(Reading database ... 287446 files and directories currently installed.)
Purging configuration files for nexus-wallet (1.0.1) ...
```

**Comment by pioruns** [comment]
Just installed .deb in Linux Mint VirtualBox machine. It opens a window this time. Stuck on "Loading wallet, please Wait...", just like in issue #47.

Terminal:

```
$ nexus_wallet 
[08:33:05.615] [info] File server listening on port 9331!
Error reading file: settings.json => Error: ENOENT: no such file or directory, open '/home/pioruns/.config/Nexus_Wallet/settings.json'
Error reading file: settings.json => Error: ENOENT: no such file or directory, open '/home/pioruns/.config/Nexus_Wallet/settings.json'
[08:33:05.718] [info] Checking if core binary exists: /opt/Nexus Wallet/resources/app/cores/nexus-linux-x64
[08:33:05.718] [info] Core binary exists
[08:33:05.719] [info] Core Manager: Data Directory path not found. Creating folder: /home/pioruns/.Nexus
[08:33:05.720] [info] Core Parameters: -rpcport=9336,-datadir=/home/pioruns/.Nexus,-daemon,-avatar,-server,-rpcthreads=4,-fastsync,-verbose=2,-rpcallowip=127.0.0.1,-stake=1
[08:33:05.720] [info] Core Manager: Starting core
[08:33:05.723] [info] Core Manager: Core has started (process id: 3713)
Error reading file: settings.json => Error: ENOENT: no such file or directory, open '/home/pioruns/.config/Nexus_Wallet/settings.json'
[08:33:07.131] [info] Checking for update
[08:33:07.138] [error] Error: Error: ENOENT: no such file or directory, open '/opt/Nexus Wallet/resources/app-update.yml'
    at readFile (/opt/Nexus Wallet/resources/app.asar/webpack:/node_modules/electron-updater/out/AppUpdater.js:625:1)
    at Generator.next (<anonymous>)
From previous event:
    at t.AppImageUpdater.loadUpdateConfig (/opt/Nexus Wallet/resources/app.asar/webpack:/node_modules/electron-updater/out/AppUpdater.js:620:1)
    at t.Lazy.loadUpdateConfig (/opt/Nexus Wallet/resources/app.asar/webpack:/node_modules/electron-updater/out/AppUpdater.js:221:1)
    at t.Lazy.creator [as value] (/opt/Nexus Wallet/resources/app.asar/webpack:/node_modules/lazy-val/out/main.js:17:1)
    at value (/opt/Nexus Wallet/resources/app.asar/webpack:/node_modules/electron-updater/out/AppUpdater.js:503:1)
    at Generator.next (<anonymous>)
    at runCallback (timers.js:694:18)
    at tryOnImmediate (timers.js:665:5)
    at processImmediate (timers.js:647:5)
From previous event:
    at t.AppImageUpdater.getUpdateInfoAndProvider (/opt/Nexus Wallet/resources/app.asar/webpack:/node_modules/electron-updater/out/AppUpdater.js:499:1)
    at getUpdateInfoAndProvider (/opt/Nexus Wallet/resources/app.asar/webpack:/node_modules/electron-updater/out/AppUpdater.js:532:1)
    at Generator.next (<anonymous>)
From previous event:
    at t.AppImageUpdater.doCheckForUpdates (/opt/Nexus Wallet/resources/app.asar/webpack:/node_modules/electron-updater/out/AppUpdater.js:529:1)
    at t.AppImageUpdater.doCheckForUpdates [as checkForUpdates] (/opt/Nexus Wallet/resources/app.asar/webpack:/node_modules/electron-updater/out/AppUpdater.js:361:1)
    at callFunction (/opt/Nexus Wallet/resources/electron.asar/browser/rpc-server.js:251:24)
    at /opt/Nexus Wallet/resources/electron.asar/browser/rpc-server.js:377:10
    at EventEmitter.ipcMain.on.args (/opt/Nexus Wallet/resources/electron.asar/browser/rpc-server.js:274:21)
    at EventEmitter.emit (events.js:182:13)
    at WebContents.<anonymous> (/opt/Nexus Wallet/resources/electron.asar/browser/api/web-contents.js:356:21)
    at WebContents.emit (events.js:182:13)
Error reading file: settings.json => Error: ENOENT: no such file or directory, open '/home/pioruns/.config/Nexus_Wallet/settings.json'
```

**Comment by pioruns** [comment]
Hello, any one? Tritium wallet on Linux = 100% doesn't work as of now. Has any testing on simplest  VM been done on this? I'd like to help, provide logs by no one is even replying here.

**Comment by VidereLicet** [comment]
This is a known issue we are addressing with 3.0.2, it works on most Linux distributions but on certain versions of Ubuntu the Daemon fails to load.

**Comment by KenCorma** [comment]
Most of the team was celebrating 4th of July so didn't get a change to dive into it. 

Have you tried to  use the appimage or cloning the repo?

**Comment by pioruns** [comment]
Hi, thanks for your replies. 
Yes I've tried AppImage please see other bug report. 
Both install versions don't work on Linux Mint (Ubuntu) and MX Linux (Debian). No need for cloning source, I just wanted to let you know about binaries you compiled for people (completely unusable on one of the most popular systems our there Ubuntu and Debian).

**Comment by pioruns** [comment]
It's been 13 days since bug report, any news? New Nexus wallet is completely broken on Linux.

**Comment by Qrysto** [comment]
@KenCorma Could you check this? I'm having some issues on my Ubuntu VM so I can't check it now.

**Comment by Qrysto** [comment]
@pioruns This issue will likely be resolved with the new core v3.0.2 which will be included in the next wallet release, expected to be out this week. Please stay tuned for the update
